addons/spectator/script_component.hpp

```diff
@@ -50,8 +50,7 @@
 #define DISTANCE_NAMES_SQR 30625 // Names are rendered within 175m, squared for `distanceSqr` speed
 #define NAME_MAX_CHARACTERS 17
 
-#define LIST_ENTITIES localize "STR_A3_Spectator_Entities"
-#define LIST_LOCATIONS localize "STR_A3_Spectator_Locations"
+#define LIST_ENTITIES 0
```

Comment:
Nah, it was only ever used to check the current tab (and localised for some reason). So I may as well just use the tab indices.